locales/de.json

```diff
@@ -0,0 +1,623 @@
+    },
+    "aria-required-parent": {
+      "description": "",
+      "help": "Bestimmte ARIA-Rollen müssen in spezifischen, übergeordneten Eltern-Rollen enthalten sein."
+    },
+    "aria-roles": {
+      "description": "",
+      "help": "Verwendete ARIA-Rollen müssen gültigen Werten entsprechen."
+    },
+    "aria-valid-attr-value": {
+      "description": "",
+      "help": "Verwendete ARIA-Attribute müssen gültigen Werten entsprechen."
+    },
+    "aria-valid-attr": {
+      "description": "",
+      "help": "Verwendete ARIA-Attribute müssen gültigen Namen entsprechen."
+    },
+    "audio-caption": {
+      "description": "",
+      "help": "<audio>-Elemente müssen eine Untertitelung (captions track) besitzen."
+    },
+    "blink": {
+      "description": "",
+      "help": "<blink>-Elemente sind veraltet und dürfen nicht verwendet werden."
+    },
+    "button-name": {
+      "description": "",
+      "help": "Schaltflächen müssen wahrnehmbaren Text enthalten."
+    },
+    "bypass": {
+      "description": "",
+      "help": "Die Seite muss Wege bereitstellen, um mehrfache Blöcke zu übergehen."
+    },
+    "checkboxgroup": {
+      "description": "",
+      "help": "Checkbox-Eingaben mit demselben Namens-Attributwert müssen Teil einer Gruppe sein."
+    },
+    "color-contrast": {
+      "description": "",
+      "help": "Elemente müssen einen ausreichenden Farbkontrast haben."
+    },
+    "definition-list": {
+      "description": "",
+      "help": "<dl>-Elemente dürfen unmittelbar nur korrekt verschachtelte <dt>- und <dd>-Gruppen, <script>- oder <template>-Elemente enthalten."
+    },
+    "dlitem": {
+      "description": "",
+      "help": "<dt>- und <dd>-Elemente müssen in einem <dl>-Element enthalten sein."
+    },
+    "document-title": {
+      "description": "Stellt sicher, dass jedes HTML-Dokument einen nicht leeres <title>-Element besitzt.",
+      "help": "Dokumente müssen ein <title>-Element besitzen, um die Navigation zu erleichtern."
+    },
+    "duplicate-id": {
+      "description": "",
+      "help": "Der Wert des id-Attributes muss einzigartig sein."
+    },
+    "empty-heading": {
+      "description": "",
+      "help": "Überschriften dürfen nicht leer sein."
+    },
+    "focus-order-semantics": {
+    "description": "",
+    "help": "Elemente in der Fokusreihenfolge benötigen eine Rolle, die für interaktive Inhalte geeignet ist."
+    },
+    "frame-tested": {
+    "description": "",
+    "help": "Frames müssen mit axe-core getestet werden."
+    },    
+    "frame-title-unique": {
+      "description": "",
+      "help": "Frames müssen ein einzigartiges title-Attribut besitzen."
+    },
+    "frame-title": {
+      "description": "",
+      "help": "Frames müssen ein title-Attribut besitzen."
+    },
+    "heading-order": {
+      "description": "",
+      "help": "Die Reihenfolge der Überschriftsebenen sollten semantisch korrekt sein."
```

Comment:
Yes, sounds good! I'm just not sure, whether "Stufe" is a commonly used word in this context.
Maybe "Überschriftsebenen sollten nur um eine Ebene steigen."
or to avoid repition "Überschriftsebenen sollten nur um eins steigen." ?

Reply:
I like the last one `Überschriftsebenen sollten nur um eins steigen`